<launch>

  <node pkg="remyJr" name="brain1" type="brain.py" respawn="true"/>
  <node pkg="remyJr" name="sensor_monitor" type="sensor_monitor.py" required="true"/>
  <node pkg="remyJr" name="graphs" type="graph_motion_data.py" respawn="true" />

</launch>
